README.md

```diff
@@ -25,14 +38,6 @@ assert!(unicode_blocks::is_cjk_block(unicode_blocks::CJK_UNIFIED_IDEOGRAPHS));
 assert!(unicode_blocks::is_cjk('。'));
 ```
 
-## Crates.io
-
-https://crates.io/crates/unicode-blocks
-
-## Documentation
-
-https://docs.rs/unicode-blocks
-
```

Comment:
All my crates on crates.io have them in `README.md` adn therefore I don't want to remove them.